src/software/thunderscope/thunderscope.py

```diff
@@ -83,12 +88,14 @@
     # Configure Docks
     dock_area.addDock(field_dock, "left")
     dock_area.addDock(log_dock, "bottom", field_dock)
+    dock_area.addDock(chicker_dock, "right", field_dock)
```

Comment:
you can remove this line for now since we have validated that it works, in my next PR ill add it into robot diagnostics tscope instance

Reply:
done. I also added a `setup_chicker_widget` function. Do I need to change anything else in the `thunderscope.py` file.